src/wobbly/main.py

```diff
@@ -53,7 +53,8 @@ async def lifespan(app: FastAPI) -> AsyncIterator[None]:
 
 # Attach the routers.
 app.include_router(internal.router)
-app.include_router(service.router, prefix=f"{config.path_prefix}")
+app.include_router(admin.router, prefix=config.path_prefix)
```

Comment:
Should this be mounted at `f'{config.path_prefix}/admin`? Right now, we could protect the `/services` path prefix, but:
* If you like the "don't use a hierarchy" suggestion, this won't work
* If we add other admin routes that don't have a `/service` prefix, it could be easy to forget to update the admin ingress to protect them.